src/lib/nexus-schema-stateful/stateful-nexus-schema.ts

```diff
@@ -28,6 +29,30 @@ export interface NexusSchemaStatefulBuilders {
   idArg: typeof NexusSchema.idArg
   extendType: typeof NexusSchema.extendType
   extendInputType: typeof NexusSchema.extendInputType
+  /**
+   * Add a GraphQL.js scalar type and expose it as a method in your definition builders.
+   * 
+   * @example
+   * 
+   * ```ts
+   * import { schema } from 'nexus'
+   * import { GraphQLDate } from 'graphql-iso-date'
+   *
+   * schema.addScalarAsMethod(GraphQLDate, 'date')
+   *
+   * schema.objectType({
+   *  name: 'SomeObject',
+   *  definition(t) {
+   *   t.date('createdAt') // t.date() is now available (with types!) because of `addScalarAsNexusMethod`
+   *  },
+   * })
+   * ```
+   */
+  addScalarAsMethod: typeof NexusSchema.asNexusMethod
```

Comment:
Simply `schema.scalar(...)` + your great jsdoc 🤔

Reply:
But we already have `schema.scalarType` which makes `schema.scalar` quite confusing doesn't it ?